### ISSUE TITLE
Terms of service and privacy policy did not show on email auth UI process.

### ISSUE THREAD

**Issue by shortstopyomin** [firebase/FirebaseUI-Android#1090]
FirebaseUI-Android (Version: 3.1.2) sample code without changing any source code.

Terms of service and privacy policy did not show on email auth UI process.

master branch.

```
public void signIn(View view) {
        startActivityForResult(
                AuthUI.getInstance().createSignInIntentBuilder()
                        .setTheme(getSelectedTheme())
                        .setLogo(getSelectedLogo())
                        .setAvailableProviders(getSelectedProviders())
                        .setTosUrl(getSelectedTosUrl())
                        .setPrivacyPolicyUrl(getSelectedPrivacyPolicyUrl())
                        .setIsSmartLockEnabled(mEnableCredentialSelector.isChecked(),
                                mEnableHintSelector.isChecked())
                        .setAllowNewEmailAccounts(mAllowNewEmailAccounts.isChecked())
                        .build(),
                RC_SIGN_IN);
    }
```

**Comment by Kurt29** [comment]
I am not sure and cannot try it for my own, but can you try the same code withouts setting a custom theme?

**Comment by samtstern** [comment]
@shortstopyomin could you please show a screenshot of what you see when you expect to see the TOS?

**Comment by shortstopyomin** [comment]
@samtstern 

![screenshot_20180104-190103](https://user-images.githubusercontent.com/1888742/34639684-99324dcc-f31f-11e7-9944-f2c4359b6bd7.jpg)

**Comment by samtstern** [comment]
@shortstopyomin thanks for the screenshot, I have sent a PR to fix this #1094

**Comment by borrom-dev** [comment]
I'm quite new to open source project, but I was wondering when this fix will available?

**Comment by samtstern** [comment]
@borrom-dev we're hoping to release `3.2.0` soon, probably early next week.

**Comment by samtstern** [comment]
This has been fixed and released in version `3.2.0`.
